config/locales/application_form.yml

```diff
@@ -843,26 +843,26 @@ en:
             english_gcses:
               blank: Select at least one GCSE
             grade_english_single:
-              blank: Enter your grade
-              invalid: Enter a real grade
+              blank: Enter your English (Single award) grade
```

Comment:
Probably need to ensure consistency with science validation messages. I think @paulrobertlloyd wanted ‘science’ dropped from each as this was deemed unnecessary.

Science validation messages - https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/config/locales/application_form.yml#L811

Reply:
I wonder if English GCSEs are a bit different? 

I don't think people reference to Literature or Language GCSEs rather than English Literature or English Language. I'd be keen to see what Paul thinks here.

Reply:
There is also English single and double as well as English Studies single and double, so I think it'd be clearer to leave the extra clarification in here.